cmds/core/gopher/main.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019-2019 the u-root Authors. All rights reserved
```

Comment:
I don't think this should be a core command. It's just going to bloat up the image. It's possible we need a new command class, maybe "art" or some such.